tensorboard/plugins/projector/vz_projector/vz-projector-projections-panel.ts

```diff
@@ -132,7 +141,10 @@ export class ProjectionsPanel extends ProjectionsPanelPolymer {
         this.querySelector('#learning-rate-slider') as HTMLInputElement;
     this.superviseFactorInput =
         this.querySelector('#supervise-factor-slider') as HTMLInputElement;
-    this.iterationLabel = this.querySelector('.run-tsne-iter') as HTMLElement;
+    this.iterationLabelTSNE = this.querySelector('.run-tsne-iter') as HTMLElement;
+    this.umapNeighborsInput =
+        this.querySelector('#umap-neighbors-slider') as HTMLInputElement;
+    this.runUmapButton = this.querySelector('.run-umap') as HTMLButtonElement;
```

Comment:
Nope. Just following the (confusing and inconsistent) patterns from the rest of the app... I think it might be worth a full-pass comb-through of a bunch of these inconsistencies around how the polymer part of the app is implemented, and I'd be happy to do that in another PR